tests/Samples.Server.Tests/ResponseTests.cs

```diff
@@ -103,7 +104,7 @@ public async Task Batched_Query_Should_Return_Multiple_Results()
                 HttpMethod.Post,
                 new StringContent("Oops", Encoding.UTF8, "application/json"),
                 HttpStatusCode.BadRequest,
-                "JSON body text could not be parsed. Body text should start with '{' for normal graphql query or with '[' for batched query."
+                "JSON body text could not be parsed. 'O' is an invalid start of a value. Path: $ | LineNumber: 0 | BytePositionInLine: 0."
```

Comment:
This error message (when parsing "Oops") is now just what was returned directly from the System.Text.Json deserializer instead of a custom message.

Reply:
ok